js/modules/k6/experimental/csv/module.go

```diff
@@ -0,0 +1,238 @@
+// Package csv provides a CSV parser for k6.
+package csv
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"sync/atomic"
+
+	"gopkg.in/guregu/null.v3"
+
+	"go.k6.io/k6/js/promises"
+
+	"go.k6.io/k6/js/modules/k6/experimental/fs"
+
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modules"
+)
+
+// TODO: Should we have an option to cycle through the file (restart from start when EOF is reached, or done is true)?
+// TODO: Should we have an option to skip empty lines (lines with no fields?)
+// TODO: Should we have an option to skip lines based on a specific predicate func(linenum, fields): bool ?:w
```

Comment:
Again my opinion; I'd say that'd be a nice-to-have, but not critical for a first iteration, so I'd keep that idea for a second iteration.

Reply:
Talking with Pawel, he agreed with you, and mentioned that our current scope is good enough for 90% of the use cases, and we could do it in the future if we still want to 👍🏻